### PR TITLE
5293-Remove-eventlistener-on-microcart-destroyed-hook:

### PR DESCRIPTION
**Pull request by bohdan-karapet**
- refactored addeventlistener
- added method that load cart if tab is visible
- removed eventlistener
- updated changelog

### Related Issues
#5293

closes #

### Short Description and Why It's Useful
Not sere if it's useful.
On new version it was fixed by moving functionality to theme (https://github.com/DivanteLtd/vsf-default/blob/master/components/core/blocks/Header/MicrocartIcon.vue#L28)
But on 1.11 it's causing issues.
Also can be useful for themes, based on old default theme.

### Upgrade Notes and Changelog
- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Display default placeholder on the Product page (#5088)
+- Removed `visibilitychange` eventlistener on microcart beforeDestroy hook
```
Comment (Fifciu) [inline CHANGELOG.md:44]: Add ID of the issue. Bestly, with the link to the issue.

**Review by Fifciu — CHANGES_REQUESTED** [review]
Please update the changelog and resolve conflicts

File: CHANGELOG.md (hunk 1 above)
Comment (bohdan-karapet) [reply CHANGELOG.md]: Done
Sorry for delay)

**Review by Fifciu — APPROVED** [review]

**Comment by Fifciu** [comment]
Good job, thanks!

**Pull request merged**